rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -197,7 +197,7 @@ components:
           type: string
           format: date-time
           description: The server time when the error was generated.
-          example: 2019-05-09T14:47:13.101+0000
+          example: 2019-05-09T14:47:13.101+00:00
```

Comment:
❓The example for Last-Modified is `2017-06-11T14:32:28Z`, but the example for timestamp is `2019-05-09T14:47:13.101+00:00`, even though they both have `format: date-time`. Are both of these examples valid even though they appear to have a different format?

Reply:
Yes according to https://tools.ietf.org/html/rfc3339#section-5.6
```   
   time-hour       = 2DIGIT  ; 00-23
   time-minute     = 2DIGIT  ; 00-59

   time-numoffset  = ("+" / "-") time-hour ":" time-minute
   time-offset     = "Z" / time-numoffset